feature-libs/cart/base/core/facade/active-cart.service.ts

```diff
@@ -64,9 +66,14 @@ export class ActiveCartService implements ActiveCartFacade, OnDestroy {
     switchMap((cartId) => this.multiCartFacade.getCartEntity(cartId))
   );
 
+  // Flag to prevent cart loading when logged in with code flow
+  // Instead of loading cart will run loadOrMerge method
+  protected shouldLoadCartOnCodeFlow = true;
+
   constructor(
     protected multiCartFacade: MultiCartFacade,
-    protected userIdService: UserIdService
+    protected userIdService: UserIdService,
+    @Optional() protected winRef?: WindowRef // This dependency might be required in the next major release.
```

Comment:
I think we can remove `@Optional` decorator. What do you think about this: 
```
    /**
     * TODO: (From 7.0) This dependency should be required from the next major.
     * It is used to fix a problem related to merge carts in oAuth redirect flow.
     *
     * For more context please see: CXSPA-617.
     */
    protected winRef?: WindowRef
  ```

Reply:
Changed, thanks